presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -196,6 +198,10 @@ public static DistributedQueryRunner createQueryRunner(
                 .putAll(extraProperties)
                 .build();
 
+        // This function is used when initializing TPC-H tables. Execute it while nodes are starting
+        // to lower the wall time for table copies during startup
+        CompletableFuture.runAsync(TextPool::getDefaultTestPool);
```

Comment:
Why does the iceberg queryrunner have the extra tpcds check?

Reply:
Good catch. I was trying to prevent wasting CPU cycles on this execution when the flag is off. I will add it here.